msprime/__init__.py

```diff
@@ -51,3 +51,4 @@
 from msprime.exceptions import *
 from msprime.mutations import *
 from msprime.likelihood import *
+from msprime.species_tree import *
```

Comment:
Not sure we want to import everything into the msprime namespace here, as there's going to be quite a bit of private stuff in here we don't want the user to see. I think we should import ``parse_species_tree`` and parse_startbeast`` here.

Reply:
Makes sense!

Reply:
Now changed.